pkg/util/certUtil.go

```diff
@@ -0,0 +1,104 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package util
+
+import (
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/sha256"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/hex"
+	"encoding/pem"
+	"fmt"
+	"math/big"
+	"time"
+)
+
+func Checksum(src []byte) string {
+	h := sha256.New()
+	h.Write(src)
+	out := h.Sum(nil)
+	return hex.EncodeToString(out)
+}
+
+func GenerateRSAKeyPair(namespace, serviceName string) ([]byte, []byte, error) {
```

Comment:
CertUtil help us generate CA Cert for a certain service with ten years default validity period